src/components/cars/carslist.js

```diff
@@ -1,34 +1,35 @@
 import React, { Component } from 'react';
 import Car from './car';
-import Pagination from '../dataTable/pagination';
+import { connect } from 'react-redux';
+import { loadCars } from '../../actions/actionCars'
```

Comment:
don't use such naming
actions/ationsCars
actions/cars is enought or
you can use something like
actions/cars.action.js